streamlined text descriptions (#63)
uaenvironment/uaenvironment.github.io: README.md

## Overview

### Who we are

> The Technical Research Applications Development (TRAD) group is found within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). TRAD is a group of technology research professionals and students who provide a variety of technical research services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within AIR. Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between.
> 

Reviewer: ```suggestion
The Technical Research Applications Development (TRAD) is a group of technology research professionals and students who provide technical research services and collaborations to the [Centers and Programs](https://air.arizona.edu/centers-programs) found within within the [Arizona Institute for Resilience (AIR)](https://air.arizona.edu). Our project collaborations span the globe and make meaningful impacts from the desert southwestern United States to the Middle East and places in between.

```
 Maybe a nit, but I felt like the first sentence didn't say anything previously. If we're trying to answer the question "What is TRAD?" answering with "TRAD is in AIR" first is a little strange. I think this rework gets to the point a little faster.  What do you think?

> TRAD is led by [Rey Granillo](https://github.com/reyg3) Director of Technology and R&D Engineering, [Leland Boeman](https://github.com/lboeman) Research & Development Software Engineer, and [Thomas Weiss](https://github.com/tweissaz) Research & Development Systems Engineer.

Reviewer: It feels a little weird to me to abbreviate R&D here  in the first title, then spell it out for Thomas and I.

Author: Good catch. Removed the abbreviation for consistency.

### Our proficiencies
- Research & Development / Proof of Concept

>   - Research focusd software development

Reviewer 2: ```suggestion
  - Research-focused software development
```

  - Hardware and sensor development
- Cloud computing infrastructure
  - Amazon Web Services (AWS)
  - Google Cloud Platform (GCP)
- Data assimilation, processing, and presentation
- Database architecture and development
- Application Programming Interface (API) development
- Remote sensing 
  - LoRaWAN
  - Microcontroller design and development
- 3D printing
- Machine Learning (ML)

### How we do it
#### Development Process

> By developing and combining technologies, we incorporate novel technical solutions into new and existing research. Using a collaborative model, we can better understand project data, the research questions being asked, and provide higher quality research collaborations. Having that greater understanding has led us to find correlations in data that were not previously realized which have led to new research project ideas and proposals. We collaborate with researchers during the proposal process and can provide a proof-of-concept application or tool showing what is technically feasible if project funding is awarded. Upon project completion, we aid with the technical writing of publications which outline our findings and technical solutions that were implemented.

Reviewer: ```suggestion
Using a collaborative model, TRAD members gain a broader understanding of the research questions being asked, and collaborators gain a better understanding of how cutting edge technology can be applied to their area of expertise. These collaborations have uncovered correlations in data that were not previously realized which have led to new research project ideas and proposals. We collaborate at each step of the research process from proposal development to publication. TRAD can provide a proof-of-concept application or tool showing what is technically feasible to add a competitive edge to proposals. Upon project completion, we aid with the technical writing of publications which outline our findings and technical solutions that were implemented.
```
What do you think about this rework? I felt like some of the points read a little awkwardly and tried to make it more direct.

  - Majors: Ecology & Evolutionary Biology and Optical Sciences & Engineering

## Research Collaborators

> In addition to our core research team, we've had a number of collaborators across AIR programs and affiliated groups across the University of Arizona. 

Reviewer 2: Maybe with some upcoming projects we also mention outside collaborators?

Author: I think that's a good idea. We can list ALERTWest and B2. Are there others you can think of that we should highlight @tweissaz ?

Reviewer 2: Nah, all I can think of for now.
